tasks/prelim.yml

```diff
@@ -59,7 +59,9 @@
   apt:
     name: "{{ auditd_package[ansible_os_family] }}"
     state: present
-    install_recommends: false
```

Comment:
Why install recommends? Ff possible I want to avoid this because otherwise unnecessary packages will end up on the system. If you need a package please specify it explicitly